shared/engine/transport.native.js

```diff
@@ -1,37 +0,0 @@
-// @flow
-import BaseTransport from './rpc'
-import {Buffer} from 'buffer'
-
-class MobileTransport extends BaseTransport {
-  // $FlowIssue
-  constructor (incomingRPCCallback, rpcWriteCallback) {
```

Comment:
we got but ignored the connect callback so theoretically the _onConnect would be ignored. this was all moved into the SharedTransport layer so its the same